middleware/transaction.go

```
@@ -0,0 +1,49 @@
+			c.JSON(500, gin.H{"error": "トランザクションの開始に失敗しました"})
+			c.Abort()
+			return
+		}
+
+		// トランザクションをコンテキストに保存
+		c.Set("tx", tx)
+
+		// パニックをキャッチしてロールバック
+		defer func() {
+			if r := recover(); r != nil {
+				tx.Rollback()
```

golangci-lint (1.64.8)

Error return value of `tx.Rollback` is not checked

(errcheck)